pkg/genericapiserver/server/genericapiserver.go

```diff
@@ -59,6 +59,10 @@ type APIGroupInfo struct {
 	// If nil, defaults to groupMeta.GroupVersion.
 	// TODO: Remove this when https://github.com/kubernetes/kubernetes/issues/19018 is fixed.
 	OptionsExternalVersion *schema.GroupVersion
+	// MetaGroupVersion defaults to "meta.k8s.io/v1" and is the scheme group version used to decode
+	// common API implementations like ListOptions. Future changes will allow this to vary by group
+	// version (for when the inevitable meta/v2 group emerges).
+	MetaGroupVersion *schema.GroupVersion
```

Comment:
I *think* this would be used for other meta related behaviors.  Basically it's asserting the server group version (which was the old name) which influences default behavior.  And should be reported in swagger.